scripts/internal/manylinux-build-wheels.sh

```diff
@@ -67,12 +67,19 @@ for PYBIN in "${PYBINARIES[@]}"; do
     echo "PYTHON_LIBRARY:${PYTHON_LIBRARY}"
 
     ${PYBIN}/pip install -r /work/requirements-dev.txt
+    build_path=/work/ITK-$(basename $(dirname ${PYBIN}))-manylinux1_${arch}
+    # Clean up previous invocations
+    rm -rf $build_path
     ${PYBIN}/python setup.py bdist_wheel --build-type MinSizeRel -G Ninja -- \
       -DITK_SOURCE_DIR:PATH=/work/standalone-${arch}-build/ITK-source \
+      -DITK_BINARY_DIR:PATH=${build_path} \
       -DPYTHON_EXECUTABLE:FILEPATH=${PYTHON_EXECUTABLE} \
       -DPYTHON_INCLUDE_DIR:PATH=${PYTHON_INCLUDE_DIR} \
       -DPYTHON_LIBRARY:FILEPATH=${PYTHON_LIBRARY}
     ${PYBIN}/python setup.py clean
+    find $build_path -name '*.cpp' -delete -o -name '*.xml' -delete
+    rm -rf $build_path/Wrapping/Generators/castxml*
+    find $build_path -name '*.o' -delete
```

Comment:
Based on your conversation, this is done here to reduce the size of the packaged archive. 

May be a comment could be added ?